src/commands/game_commands/play.ts

```diff
@@ -145,6 +158,29 @@ export default class PlayCommand implements BaseCommand {
                     "command.play.help.example.suddenDeath",
                 ),
             },
+            {
+                example: `${clickableSlashCommand(
+                    COMMAND_NAME,
+                    GameType.CLIP,
+                )} timer:0.75`,
+                explanation: i18n.translate(
+                    guildID,
+                    "command.play.help.example.clip",
+                    {
+                        clipTimer: "`0.75`",
+                    },
+                ),
+            },
+            {
+                example: clickableSlashCommand(COMMAND_NAME, GameType.CLIP),
+                explanation: i18n.translate(
+                    guildID,
+                    "command.play.help.example.clip",
+                    {
+                        clipTimer: `\`${CLIP_DEFAULT_TIMER}\``,
+                    },
+                ),
+            },
```

Comment:
What's teh diff between these

Reply:
This example showcases an argument being passed to `/play clip`